ClientApp/viking-vault/src/AdminDashboard/UserData.tsx

```diff
@@ -239,8 +241,8 @@ class UserData extends React.Component<IUserDataProp, IPageState>{
         return( 
             <div className = "user-container"> 
                 <div className = "user-container-inside">
-                    <div className = "img-container">
-                         {this.state.user.pictureLink === "" ? <img src = {DefaultProfilePicture} className = "profile-img"/> : <img src = {this.state.user.pictureLink} className = "profile-img"/> }                 
+                    <div className = "img-container">  
+                        <UserIconAvatar pictureUri={this.state.user.pictureLink} pictureStyle="profile-img"  defaultPicture={DefaultProfilePicture}/>
```

Comment:
`pictureStyle` seems to me like it's only a class name, not a style

Reply:
Yes, but technically it is the class name of the **_style_** we want for our tags :3